unittests/webserver/ServerTest.cpp

```diff
@@ -191,15 +191,19 @@ class ExceptionExecutor
 TEST_F(WebServerTest, Http)
 {
     auto e = std::make_shared<EchoExecutor>();
-    auto const server = Server::make_HttpServer(cfg, ctx, std::nullopt, dosGuard, e);
+    auto server = std::shared_ptr<Server::HttpServer<EchoExecutor>>();
+    boost::asio::dispatch(
```

Comment:
This looks like a race-condition. It depends what the `syncPost` will do if the server is not yet up. Worth checking 👍

Reply:
Added condition variable